org.syphon.Syphon.json

```diff
@@ -0,0 +1,97 @@
+{
+    "app-id": "org.syphon.Syphon",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "41",
+    "sdk": "org.gnome.Sdk",
+    "command": "syphon",
+    "separate-locales": false,
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--socket=pulseaudio",
+        "--share=network",
+        "--device=all",
+        "--filesystem=xdg-download",
+        "--talk-name=org.freedesktop.Notifications",
+        "--own-name=org.syphon.Syphon"
+    ],
+    "modules": [
+        {
+            "name": "libjsoncpp",
+            "buildsystem": "cmake",
+            "config-opts": [
+                "-DCMAKE_BUILD_TYPE=release",
```

Comment:
```suggestion
                "-DCMAKE_BUILD_TYPE=RelWithDebInfo",
```

Reply:
Out of curiosity, why this change?

Reply:
Debug info gets automatically removed from the release that most users get, but it's packaged in a .Debug extension and can be added back in to help make development and bug fixing easier.

Reply:
Ah, okay! :)